src/Entity/Conference.php

```diff
@@ -399,4 +376,16 @@ public function isOnline(): bool
     {
         return $this->online;
     }
+
+    public function addTag(string $tag): self
+    {
+        $this->tags[] = $tag;
+
+        return $this;
+    }
+
+    public function getTags(): array
+    {
+        return $this->tags;
+    }
```

Comment:
Is not a `removeTag` method missing here?

Reply:
Conference are readonly IIRC. So let's not add not used code

Reply:
Oh yes, you're right. My bad